api/client/stats_helpers.go

```diff
@@ -24,6 +24,7 @@ type containerStats struct {
 	NetworkTx        float64
 	BlockRead        float64
 	BlockWrite       float64
+	PidsCurrent      uint64
```

Comment:
I can't find that same function on master the stats one in daemon?

On Tuesday, March 8, 2016, Tõnis Tiigi <notifications@github.com> wrote:

> In api/client/stats_helpers.go
> <https://github.com/docker/docker/pull/18697#discussion_r55461314>:
>
> > @@ -24,6 +24,7 @@ type containerStats struct {
> >  	NetworkTx        float64
> >  	BlockRead        float64
> >  	BlockWrite       float64
> > +	PidsCurrent      uint64
>
> Where is this being set? I think we need something like this: 103cc23
> #diff-234033fe3b8730c35e53ce0d8dbda485R119
> <https://github.com/docker/docker/commit/103cc2369e8c5d6e7264552a7e7f7f990aba1417#diff-234033fe3b8730c35e53ce0d8dbda485R119>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/18697/files#r55461314>.
>


-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>